quupe-RN/js/screens/SingleItem/SingleItemContainer.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import gql from 'graphql-tag';
+import { Mutation } from 'react-apollo';
+import SingleItem from './SingleItem';
+
+const mockItemData = {
+    title: 'Ski - Woman 160cm',
+    price: '12',
+    priceOneWeek: '50',
+    priceOneMonth: '180',
+    id: 'cjifgp62s3qhb0174jbmp0oa2',
+    owner: {
+        fullname: 'Andrew',
+        profileImage:
+            'https://hips.hearstapps.com/sev.h-cdn.co/assets/16/17/1024x762/gallery-1461600984-screen-shot-2016-04-25-at-121524-pm.jpg?resize=768:*',
+        id: 'cjig7qmm227kd0122oypew1ou'
+    },
+    image:
+        'https://images-na.ssl-images-amazon.com/images/I/51XorXpUSDL._SX425_.jpg',
+    category: 'Outdoor',
+    year: '2017',
+    condition: 'New',
+    description: 'These are some cool skis',
+    latitude: 40.593374,
+    longitude: -111.8905171
+};
+
+const sendMessageMutation = gql`
+    mutation createRoom($peopleIds: [ID!], $messages: [RoommessagesMessages!]) {
+        createRoom(peopleIds: $peopleIds, messages: $messages) {
+            id
+            messages {
+                text
+            }
+        }
+    }
+`;
+
+class SingleItemContainer extends Component {
+    constructor() {
+        super();
+        this.state = {
+            currentTab: 'Lender Info',
+            sentMessage: false
```

Comment:
We have to find another way to do this since we should not be using the component state.